### ISSUE TITLE
FPTaylor binary missing for macos and windows

### ISSUE THREAD

**Issue by elmisback** [herbie-fp/odyssey#55]
I believe this is blocked on FPTaylor not building properly on Windows. There was some plan to switch to FPTaylorJS.

We may be able to support on macos at least for now, but I vaguely recall another issue with that build as well. Need to test and report here.

**Comment by elmisback** [comment]
I'm going to close this issue for now since we host FPTaylor on the web. Feel free to reopen if local hosting becomes important.
